lib/diagnostic.js

```diff
@@ -36,7 +69,17 @@ const newArray = function newArray(length, defaultValue) {
 // The function you write should return
 // the object passed in as an argument or the object created
 const addMethod = function addMethod(propertyName, method, obj) {
+  let newObj = {};
+  newObj = obj;
```

Comment:
Not quite. If no object is passed in through `obj`, this will set `newObj` to `undefined`.